modeling/03_Per_Sat_Model.Rmd

```diff
@@ -478,9 +508,28 @@ trainedGTB_sen = (ee.Classifier.smileGradientTreeBoost(10).train(
 ### Landsat 5
 
 ```{python}
+trainingMatrixGTB_ls5 = (trainedGTB_ls5
+  .confusionMatrix())
+
+#convert to pandas dataframe with class info
+training_conf_l5 = (pd.DataFrame(
+  trainingMatrixGTB_ls5.getInfo(),
+  index=[class_values],
+  columns =[class_values]
+  ))
+print('GTB Training Confusion Matrix for Landsat 5:')
+print(training_conf_l5)
+
+#reformat and save
+training_conf_l5['mission'] = 'Landsat 5'
+training_conf_l5.reset_index(inplace = True)
+training_conf_l5 = training_conf_l5.rename(columns = {'level_0': 'class'})  
+training_conf_l5.to_csv('data/output/GTB_'+v_date+'_L5_training_confusion.csv', index = False)
```

Comment:
totes. This is in two places, so I pushed a new commit to solve this. 

I did update the files, but they don't track as new/different, since Git doesn't care about caps in a file name, apparently.